sw/host/opentitantool/src/command/lc.rs

```diff
@@ -478,6 +478,9 @@ impl CommandDispatch for VolatileRawUnlock {
             /*use_external_clk=*/ true,
             /*post_transition_tap=*/ JtagTap::LcTap,
             &self.jtag_params,
+            // whether we expect the RAW unlock feature to be present or not.
+            // on prod silicon this should be disabled.
+            false,
```

Comment:
@timothytrippel can you take a look at this here?
we do not expect this to work on production silicon, and we need to test this because if it were supported, that would be a bad loophole.

Reply:
yep this looks good to keep the testing path open. Thanks @msfschaffner !